client/src/routes/[anime_id]/+page.server.ts

```diff
@@ -37,14 +36,49 @@ export const load = (async ({ params }) => {
 export const actions = {
     addToFavorites: async ({ request }) => {
         const form = await request.formData();
+        const requestBody = await z
+            .object({
+                mal_id: z.string(),
+                title: z.string(),
+                image: z
+                    .string()
+                    .startsWith("https://cdn.myanimelist.net/images"),
+            })
+            .safeParseAsync({
+                mal_id: form.get("mal_id"),
+                title: form.get("title"),
+                image: form.get("image"),
+            });
```

Comment:
i think you can use safeParse? ;p No need to mark it async and await it the same time